From 938a51d00bfe59a6d7b46e10d8f5f037d54cd75d Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 16 Mar 2020 05:57:37 +0200
Subject: [PATCH] [SW-0] goya2: onix: update uart baudrate

It is half for the regular rate i.e. 57600 bps and not 115200 bps

Change-Id: I36aae6667512658a4e1473eed7a12c1835eeaf7f
---

diff --git a/goya2/onix.c b/goya2/onix.c
index 026db64..21bc4ce 100644
--- a/goya2/onix.c
+++ b/goya2/onix.c
@@ -25,7 +25,7 @@
 
 void hl_uart_init(void)
 {
-	preboot_uart_init(SYS_CLOCK_FREQUENCY / HL_UART_BAUDRATE);
+	preboot_uart_init(SYS_CLOCK_FREQUENCY / (HL_UART_BAUDRATE / 2));
 }
 
 u32 hl_spi_baudr_get(void)
